.eslintrc.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    "extends": "airbnb-base"
+};
```

Comment:
Add newline? POSIX compliant

Reply:
@gulzar1996  is stuck on an error for line-breaks confusion in Windows and Linux. Any suggestions @selfup ?

Reply:
Oh interesting. Are you on Windows?

Reply:
For a project I have I just told eslint to ignore linebreak styles:

Example here: https://github.com/selfup/hyperapp-one/blob/master/.eslintrc.js#L7

Another option is to set LF instead of CRLF in your editor (if that's possible)

Reply:
Not me.. @gulzar1996 is. He pinged me with this error I never came around. :(
This one should help. Only he can test it.

Reply:
Oh ok, yea I guess adding that to the eslintrc will solve the problem 😄